prody/proteins/interactions.py

```diff
@@ -867,14 +865,31 @@ def calcProteinInteractions(atoms, **kwargs):
     return AllInteractions
 
 
-def calcHydrogenBondsDCD(atoms, trajectory, distA=3.0, angle=40, cutoff_dist=20, **kwargs):   
+def calcHydrogenBondsDCD(atoms, trajectory=None, distA=3.0, angle=40, cutoff_dist=20, **kwargs):   
     """Compute hydrogen bonds for DCD trajectory using default parameters.
         
     :arg atoms: an Atomic object from which residues are selected
     :type atoms: :class:`.Atomic`
         
-    :arg trajectory: DCD filename
-    :type trajectory: str"""
+    :arg trajectory: trajectory file
+    :type trajectory: class:`.Trajectory`
+
+    :arg distA: non-zero value, maximal distance between donor and acceptor.
+    :type distA: int, float, default is 3.0
+    
+    :arg angle: non-zero value, maximal (180 - D-H-A angle) (donor, hydrogen, acceptor).
+    :type angle: int, float, default is 40.
+    
+    :arg cutoff_dist: non-zero value, interactions will be found between atoms with index differences
+        that are higher than cutoff_dist.
+        default is 20 atoms.
+    :type cutoff_dist: int
+    
+    :arg selection: selection string
+    :type selection: str
+    
+    :arg selection2: selection string
```

Comment:
it would probably be good to explain what this is better

Reply:
I will do that. I was planning to do it in the tutorial, but I can do it here as well. 
selection is for any selection you want, such as chain A or chain B and resid 50 to 100, etc.
If you want to have interactions ONLY between two chains you can use selection and selection2:
selection='chain A', selection2='chain B'

Reply:
Not all users will read the tutorial and those that do may forget. It's still good to say that selection2 is optional and that a sphere around selection will be provided if it isn't provided.

Reply:
Done